controllers/hash.php

```diff
@@ -1,7 +1,34 @@
 <?php
+function getHash() {
```

Comment:
In my opinion this part of code is not necessary why?
First of all if you are working with files in PHP, better way is using **Iterators**. Like [DirectoryIterator](http://php.net/manual/en/class.directoryiterator.php) or [SplFileObject](http://php.net/manual/en/class.splfileobject.php)

Problem with git can be solved like this:
1. Linux → **sudo apt install git -y** ( Git problem solved. )
2. **Windows → install git core and add git to env path.** ( Git problem solved )

Better PR in my opinion:
* Create MakeFile or make.sh which all necessary dependencies  or add to Readme info about dependencies ;)
* Option 2: create virtual machine form http://phansible.com/

```sh
#!bin/bash
sudo apt update
sudo apt install  php5-common php5-intl php5  apache or nginx etc..
[…]
```